tools/requirements.txt

```diff
@@ -92,7 +91,7 @@ msgpack==1.0.2; python_version >= "3.6" and python_full_version < "3.0.0" and py
 msrest==0.6.21; python_version >= "3.6"
 mypy-extensions==0.4.3; python_version >= "3.6"
 mypy==0.812; python_version >= "3.5"
-oauth2client==4.1.3; python_version >= "3.6"
+google-auth==1.32.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6"
```

Comment:
Are you sure that running `tools/pinning/pin.sh` does not update this file?

What is the output of the command? Are you running it with the Python virtualenv activated?

Reply:
Yes I did it with venv activated. The git bash window appears for a brief time with no output.

Reply:
Don't mind I made it work! Thanks for the support!